aria/storage/sql_mapi.py

```diff
@@ -243,7 +250,10 @@ def _filter_query(self, query, filters):
     @staticmethod
     def _add_value_filter(query, filters):
         for column, value in filters.items():
-            if isinstance(value, (list, tuple)):
+            if isinstance(value, dict):
+                for predicate, operand in value.items():
+                    query = query.filter(getattr(column, predicate)(operand))
```

Comment:
well, SQL makes it fairly simple. as in `value > other_value`. but propagating it to the query is messy...